test/lib/dwolla/beneficial_owner_test.exs

```diff
@@ -1,25 +1,21 @@
 defmodule Dwolla.BeneficialOwnerTest do
-
   use ExUnit.Case
 
   import Dwolla.Factory
+  import Dwolla.TestUtils
+  import Mox
 
   alias Dwolla.BeneficialOwner
-  alias Plug.Conn
 
-  setup do
-    bypass = Bypass.open()
-    Application.put_env(:dwolla, :root_uri, "http://localhost:#{bypass.port}/")
-    {:ok, bypass: bypass}
-  end
+  setup :verify_on_exit!
 
   describe "beneficial-owners" do
-    test "get/2 requests GET and returns Dwolla.BeneficialOwner", %{bypass: bypass} do
-      body = http_response_body(:beneficial_owner, :get)
-      Bypass.expect bypass, fn conn ->
-        assert "GET" == conn.method
-        Conn.resp(conn, 200, body)
-      end
+    test "get/2 requests GET and returns Dwolla.BeneficialOwner" do
+      Dwolla.Mock
+      |> expect(:request, 1, fn :get, _, _, _, _ ->
```

Comment:
The number of expected requests is 1 by default so you could write `expect(:request, fn :get...` but it doesn't hurt to be explicit.

Reply:
Yep, it defaults to 1. I like to be explicit but probably doesn't matter too much in this case. Usually pass in the `n` to this function since, in other uses in axle pay, we do expect a different # of calls in different tests